dll/win32/shell32/wine/brsfolder.c

```diff
@@ -518,7 +531,12 @@ static void BrsFolder_CheckValidSelection( browse_info *info, LPTV_ITEMDATA lptv
     else
         EnableWindow(GetDlgItem(info->hWnd, IDC_BROWSE_FOR_FOLDER_NEW_FOLDER), TRUE);
 
+#ifdef __REACTOS__
+    if (bEnabled)
```

Comment:
When disabled?

Reply:
This fixes the correct filter that enables or disables the OK button.